src/containers/tips-library.jsx

```diff
@@ -35,6 +35,19 @@ class TipsLibrary extends React.PureComponent {
         ]);
     }
     handleItemSelect (item) {
+        /*
+            Support tutorials that require specific starter projects.
+            If a tutorial declares "requiredProjectId", check that the URL contains
+            it. If it is not, open a new page with this tutorial and project id.
+
+            TODO remove this at first opportunity. If this is still here after HOC2018,
+                 blame Eric R. Andrew is also on record saying "this is temporary".
+        */
+        if (item.requiredProjectId && window.location.href.indexOf(item.requiredProjectId) === -1) {
+            const urlParams = `?tutorial=${item.urlId}#${item.requiredProjectId}`;
+            return window.open(window.location.pathname + urlParams, '_blank');
```

Comment:
`window.open` is not good because many browsers block it as "pop-up window needs permission"

Reply:
Agreed. I think a cleaner solution would be to have a link (just a normal `<a>`) within the tutorial, saying "Open Starter Project", which would target `_blank`. I think doing that is more reliable(?) - and it's better UX anyway because they *expect* a new project to open up.

Reply:
@apple502j hmm I've tried it out on chrome and safari. In general `window.open` is allowed by browsers as long as it is in response to a direct user action (i.e. you can't just call window.open whenever you want). 

As noted in the comment, this is meant to be very temporary. But if it is being blocked by browsers, of course, then we need to revisit. We will test many different browsers shortly.